docs/sample/teflo_scenario_gchat_slack.json

```diff
@@ -0,0 +1,52 @@
+{
+	"body":
+	{
+		"scenario":
+		{
+			"name": "SCENARIO SAMPLE",
+			"overall_status": 1,
+			"executes":
+			[
+				{
+					"name": "TESTRUN SAMPLE 1",
+					"testrun_results" :
+					{
+						"aggregate_testrun_results":
+						{
+							"total_tests": 35,
+							"passed_tests": 30,
+							"failed_tests": 4,
+							"skipped_tests": 1
+						}
+					}
+				},
+				{
+					"name": "TESTRUN SAMPLE 2",
+					"testrun_results" :
+					{
+						"aggregate_testrun_results":
+						{
+							"total_tests": 30,
+							"passed_tests": 15,
+							"failed_tests": 5,
+							"skipped_tests": 10
+						}
+					}
+				}
+			],
+			"reports":
+			[
+				{
+					"provider": "reportportal",
```

Comment:
instead of provider it should be importer

Reply:
https://github.com/RedHatQE/teflo_webhooks_notification_plugin/blob/06fe5633b299405553d654aa9a0b1a8389f97620/teflo_webhooks_notification_plugin/templates/slack_template.jinja#L121

`                        {% elif (report.provider == 'reportportal' or report.importer_plugin_name == 'reportportal') and (report.import_results) %}`

it only have provider or importer_plugin_name, no 'importer'

Reply:
yes  you are correct, the provider is there for backward compatibility, we ask the users to follow the new format of using importer in the SDF instead of provider.

 so i guess for this the value should be `"importer_plugin_name": "reportportal"`. This is a property of the report resource.

Reply:
ok, got it